deploy-templates/jellyfish-product/product/deploy/kubernetes/generators/jellyfish-ui-ingress.tpl.yml

```diff
@@ -45,3 +45,14 @@ spec:
             name: jellyfish-ui
             port:
               number: 80
+  - host: '{{{JELLYFISH_LIVECHAT_HOST}}}'
```

Comment:
Ah, actually, we probably would instead want to keep the livechat ingress, but point it to the the ui pods